src/actions/users.js

```diff
@@ -166,14 +167,16 @@ function completeLogin(data) {
             return {error};
         }
 
+        const serverVersion = Client4.getServerVersion();
+
         const promises = [
-            getMyPreferences()(dispatch, getState),
-            getMyTeams()(dispatch, getState),
+            dispatch(getMyPreferences()),
+            dispatch(getMyTeams()),
         ];
 
-        const state = getState();
-        const serverVersion = Client4.getServerVersion();
-        if (!isMinimumServerVersion(serverVersion, 4, 7) && getConfig(state).EnableCustomEmoji === 'true') {
+        if (isMinimumServerVersion(serverVersion, 5, 0)) { // should probably be 5.1 depending on when the version is updated on master
+            promises.push(dispatch(getClientConfig()));
```

Comment:
I'm 0/5 on the fetching the config twice on older servers approach. I guess it'll only really affect RN so best to get the opinion of the experts on that I think.

Reply:
Maybe out of scope for this review, but I know we talked about avoiding version checks in favour of checking capability flags or some such. I'm not too concerned about double fetching, so if that got rid of the version check and complexity, I'd be in favour of that.

Reply:
Okay, I'll go change that then to keep everything simple